CHANGELOG.md

```diff
@@ -2,6 +2,119 @@
 
 ## Version 1.13.0 (unreleased):
 
+Although we originally intended 1.13.0 to be the first stable release of the metrics SDK, we've
+downgraded it to be a release candidate (RC) out of caution due to a large number of changes in both
+the metrics SDK specification and java implementation. There are several notable changes mentioned
```

Comment:
I think I might add a line before this that says "This release should be considered a release candidate for the metrics SDK."